project/profiles/capacity4more/modules/c4m/mail/c4m_mail/messages/c4m_og_invite_users/group_invite_user_subject.txt

```diff
@@ -0,0 +1 @@
+[personal:subject]
```

Comment:
@ybaras 
this one looks identical to `invite_group_member_subject.txt`  can we use the same file?